frontend/src/components/namespace/CreateNamespaceDialog.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  TextField,
+  Typography,
+} from '@mui/material';
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { post } from '../../lib/k8s/apiProxy';
+import { ActionButton } from '../common';
+
+interface CreateNamespaceButtonProps {
+  toggleFunc: () => void;
+}
+
+export function CreateNamespaceButton({ toggleFunc }: CreateNamespaceButtonProps) {
+  const { t } = useTranslation(['glossary', 'translation']);
+
+  return (
+    <ActionButton
+      color="primary"
+      description={t('translation|Create')}
+      icon={'mdi:plus-circle'}
+      onClick={() => {
+        toggleFunc();
```

Comment:
The logic for toggling the dialog for this button can be done within this button. See the DeleteButton.
We also need it visible only if the user can create namespaces (see use of AuthVisible).